addons/hr_attendance/models/res_company.py

```diff
@@ -63,8 +63,8 @@ def _init_column(self, column_name):
             self.env.cr.execute_values(query, values_args)
 
     def write(self, vals):
-        search_domain = False  # Overtime to generate
-        delete_domain = False  # Overtime to delete
+        search_domain = []  # Overtime to generate
+        delete_domain = []  # Overtime to delete
```

Comment:
Those base domains are equivalent to `TRUE_DOMAIN`.  Therefore combining them with an `OR`, those final domains are always equivalent to `TRUE_DOMAIN`, which is not what the original code meant.  So, either you initialize both to the `FALSE_DOMAIN`, or you collect all domains in lists and `OR` them at the end.